apps/ssr-tests-v9/package.json

```diff
@@ -18,7 +18,8 @@
     "test-ssr": "test-ssr \"./stories/**/*.stories.tsx\""
   },
   "dependencies": {
-    "@fluentui/react-components": "*"
+    "@fluentui/react-components": "*",
+    "@fluentui/react-utilities": "*"
```

Comment:
As the hook I'm testing is `@internal`, it is not exposed as part of `react-components`